api/types/types.go

```diff
@@ -183,3 +183,41 @@ type Workflow struct {
 	Parameters    []WorkflowParam `yaml:"parameters"`
 	Source        string          `valid:"isValidWorkflow" yaml:"source"`
 }
+
+// Node describes a node
+type Node struct {
+	UUID     string
+	Type     string
+	Name     string
+	Metadata string
+	Query    string
```

Comment:
Query has the "<action>:<query>" format, it would be clearer if we have 2 separate values (that also avoid splitting to get both values)